CHANGELOG.md

```
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `azure-pipelines.yml`
   - Remove `windows-2019` images fixes [#278](https://github.com/dsccommunity/CertificateDsc/issues/278).
   - Fix linting errors.
+### Fixed
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)